Rakefile

```
@@ -96,7 +97,7 @@ namespace :docs do
       # Primer::AvatarComponent => Avatar
       short_name = component.name.demodulize.gsub("Component", "")
 
-      File.open("docs/content/components/#{short_name}.md", "w") do |f|
+      File.open("docs/content/components/#{short_name.downcase}.md", "w") do |f|
```

```suggestion
      File.open("docs/content/components/#{short_name.underscore}.md", "w") do |f|
```

This will make `ProgressBar` -> `progress_bar` instead of `progressbar`